docs/source/tutorials/Start A New Project.md

```diff
@@ -0,0 +1,264 @@
+# Start A New Project
+
+If developers want to start a new project(Such as paper reproduction and finetune task), how can we use Libai to achieve the least code work.
+
+The advantages of using Libai to develop a task are as follows:
+- Avoid repeating a lot of work, many functions of Libai can be directly inherited in projects.
+- We can quickly reproduce the experiments we have run, because Libai automatically saves the configuration file.
+- Various information will be output during training, such as remaining training time, current iter, throughput, loss information and current learning rate.
+- Only a few parameters need to be set for distributed training.
+
```

Comment:
The advantages of using Libai to start a new project(Such as paper reproduction and finetune task) are as follows:

- Avoid redundant work, developers can directly inherit many bulit-in modules from Libai.
- Easily reproduce the experiments already run, because Libai will saves the configuration file automatically .
- Useful information will be outputed during training, such as remaining training time, current iter, throughput, loss information and current learning rate, etc.
- For distributed training, only a few parameters need to be set.

润色了一下句子

Reply:
- `Such as` -> `such as`
- `Libai` -> `LiBai`

或者改成:
The advantages of using LiBai to start a new project, e.g., paper reproduction or finetune task, are as follows:

Reply:
ok